docs/api-references/user-lifecycle-management-api.md

```diff
@@ -255,4 +255,62 @@ In addition to the [general error codes](https://developer.mastercard.com/cp-ker
 | ERROR_CODE_AUTH_METHOD_BIOMETRIC_BUT_NO_HASHES  | User found, insufficient data cannot write profile on the card – missing modalities (if any LP, RP, Face configured to the program) |
 | ERROR_CODE_INSUFFICIENT_HASHES_TO_WRITE_ON_CARD | Insufficient data cannot write hashes on the card– missing modalities (if any LP, RP, Face configured to the program)               |
 
+## 3 User Authentication
+
+### 3.1 getVerifyPasscode
+
+This API is used to verify that the Passcode provided by the user is the same as the one present on the user’s card/CP Consumer Device. in this flow, the Reliant Application takes an input from the user, which consists of 6 digits, and passes it to CPK for passcode verification.
+
+**Compatibility**
+| **Available as of CPK version #** | **Deprecated as of CPK version #** |
+|-----------------------------------|------------------------------------|
+| + CPK 2.0.1 | + n/a |
```

Comment:
The wrappers APIs such as getVerifyPasscode are available from CPK 2.4.1

Reply:
Noted. Fixed [Here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/13/commits/bb7f6408f50ee86a8a3f882ff9f4f0bcb4b2bb51)